features/maximum_coverage_drop.feature

```diff
@@ -266,3 +266,97 @@ Feature:
     And the output should contain "Line coverage has dropped by 15.22% since the last time (maximum allowed: 0.00%)."
     And the output should contain "Branch coverage has dropped by 50.00% since the last time (maximum allowed: 0.00%)."
     And the output should contain "SimpleCov failed with exit 3"
+
+  Scenario: Can set branch as primary coverage and it will fail if branch is below maximum coverage drop
```

Comment:
I think you're only missing `@branch_coverage` here and on top of all the other scenarios to make it work on JRuby :)